specification/swlifecycle.adoc

```diff
@@ -12,63 +12,59 @@ the amount of memory available on the system). To that end, the TVM should
-that point onwards until the TVM is destroyed via the sbi_tee_host_destroy_tvm 
+that point onwards until the TVM is destroyed via the `sbi_tee_host_destroy_tvm()` 
 function. 
 
 A TVM context may be created and initialized by using the 
-sbi_tee_host_create_tvm function - this global init function allocates a 
+`sbi_tee_host_create_tvm()` function - this global init function allocates a 
 set of pages for the TVM global control structure and resets the control 
 fields that are immutable for the lifetime of the TVM e.g. configuration of 
 which RISC-V CPU extensions the TVM is allowed to use, debug and pmon 
 capabilities enabled etc. 
 
-The VMM may assign memory to the TVM via a sequence of sbi_tee_host_add_tvm_page_table_pages 
-and sbi_tee_host_add_tvm_measured/zero_pages - the former grants memory pages 
-that are to contain second-stage paging structures entries 
-that translate a TVM guest physical address to the system physical address, 
-while the latter is used to hold tvm data and is referenced by the hgatp 
-leaf page table entries. For pages added to the TVM, the VMM must invoke 
-sbi_tee_host_add_tvm_measured_pages which extends the static measurement hash of the 
-TVM which will be used by the TSM to generate the attestation report 
-(evidence) when requested by a challenger (relying party). Note that if the 
-measurement steps are executed by the VMM in an incorrect order the final 
-measurements will be different and flagged during attestation. In the 
-initial set of measured TVM pages, the VMM would typically provide the 
-guest firmware, boot loader and boot kernel as well as memory needed for 
-the boot stack, heap and memory tracking structures. During 
-sbi_tee_host_add_tvm_measured/zero_pages, the memory granted is tracked by the TSM to ensure 
-that pages assigned to a TVM may not be assigned to a non-confidential VM 
-or another TVM. Memory may be lazily added to the TVM subsequent to the TVM 
-being executed using the sbi_tee_host_add_tvm_zero_pages.  
-
-Lastly, the VMM can assign memory to the TVM to hold virtual hart state via 
-sbi_tee_host_create_tvm_vcpu. Before the VMM can start 
+The VMM may assign memory to the TVM via a sequence of `sbi_tee_host_add_tvm_page_table_pages()`
+`sbi_tee_host_add_tvm_measured_pages()` and `sbi_tee_host_add_tvm_zero_pages()` - the former
+grants memory pages that are to contain second-stage paging structures entries that translate a
+TVM guest physical address to the system physical address, while the latter two are used to hold
+tvm data and is referenced by the hgatp leaf page table entries. For pages added to the TVM, the
+VMM must invoke `sbi_tee_host_add_tvm_measured_pages()` which extends the static measurement
+hash of the TVM which will be used by the TSM to generate the attestation report (evidence) when
+requested by a challenger (relying party). Note that if the measurement steps are executed by
```

Comment:
This sentence seems too big. can you break it up into smaller ones ?

Reply:
Fixed.